test/fixtures/output/flink/artifact/create.golden

```diff
@@ -0,0 +1,6 @@
++----------------+------------------+
+| Name           | my-custom-plugin |
+| Plugin ID      | ccp-123456       |
+| Version ID     |                  |
+| Content Format |                  |
```

Comment:
Why are these empty? What are the default values when you first create a flink artifact?